pom.xml

```diff
@@ -402,8 +402,8 @@ under the License.
                   <logkit>${logkit:logkit:jar}</logkit>
                   <servlet-api>${javax.servlet:servlet-api:jar}</servlet-api>
                   <commons-logging>target/${project.build.finalName}.jar</commons-logging>
-                  <commons-logging-api>target/${project.artifactId}-api-${project.version}.jar</commons-logging-api>
-                  <commons-logging-adapters>target/${project.artifactId}-adapters-${project.version}.jar</commons-logging-adapters>
+                  <commons-logging-api>target/${project.artifactId}-api-${project.version}-api.jar</commons-logging-api>
```

Comment:
That can't be right.

Reply:
I thought this was odd, could be due to changing _jarName_ to _finalName_ in the pom. However, jarName is supposed to just be an alias for finalName so it shouldn't have changed anything. When building I did see the jars being built in that naming format:

```
target/commons-logging-1.3.0-SNAPSHOT-full.jar
target/commons-logging-1.3.0-SNAPSHOT.jar
target/commons-logging-1.3.0-SNAPSHOT-sources.jar
target/commons-logging-1.3.0-SNAPSHOT-tests.jar
target/commons-logging-1.3.0-SNAPSHOT-test-sources.jar
target/commons-logging-adapters-1.3.0-SNAPSHOT-adapters.jar
target/commons-logging-api-1.3.0-SNAPSHOT-api.jar
target/commons-logging-tests.jar
```

To confirm, I just tried swapping _finalName_ back to _jarName_ but can confirm that the classifier is still appended. It is usual for maven to put the classifier at the end of the filename before the extension. 

Perhaps the better thing to do is simple allow maven to do the standard thing by not specifying a value for _jarName_ or _finalName_ in which case it'll just create:

```
target/commons-logging-1.3.0-SNAPSHOT-adapters.jar
target/commons-logging-1.3.0-SNAPSHOT-api.jar
```

However the test code itself is looking for the existence of `commons-logging-adapters-1` on the classpath.